src/C/gdal_alg.jl

```diff
@@ -869,20 +947,31 @@ Burn geometries into raster.
 * **pTransformArg**: callback data for transformer.
 * **padfGeomBurnValue**: the array of values to burn into the raster. There should be nBandCount values for each geometry.
 * **papszOptions**: special options controlling rasterization 
-"ALL_TOUCHED": 
-May be set to TRUE to set all pixels touched by the line or polygons, not just those whose center is within the polygon or that are selected by brezenhams line algorithm. Defaults to FALSE. 
-"BURN_VALUE_FROM": 
-May be set to "Z" to use the Z values of the geometries. dfBurnValue is added to this before burning. Defaults to GDALBurnValueSrc.GBV_UserBurnValue in which case just the dfBurnValue is burned. This is implemented only for points and lines for now. The M value may be supported in the future. 
-"MERGE_ALG": 
-May be REPLACE (the default) or ADD. REPLACE results in overwriting of value, while ADD adds the new value to the existing raster, suitable for heatmaps for instance.
+											
+												"ALL_TOUCHED": 
+											
+											
+												May be set to TRUE to set all pixels touched by the line or polygons, not just those whose center is within the polygon or that are selected by brezenhams line algorithm. Defaults to FALSE. 
+											
+											
+												"BURN_VALUE_FROM": 
+											
+											
+												May be set to "Z" to use the Z values of the geometries. dfBurnValue is added to this before burning. Defaults to GDALBurnValueSrc.GBV_UserBurnValue in which case just the dfBurnValue is burned. This is implemented only for points and lines for now. The M value may be supported in the future. 
+											
+											
+												"MERGE_ALG": 
+											
+											
+												May be REPLACE (the default) or ADD. REPLACE results in overwriting of value, while ADD adds the new value to the existing raster, suitable for heatmaps for instance.
```

Comment:
Not sure where the leading whitespace came from, but now it's gone after using the updated python script to generate the XML.